include/LAGraphX.h

```diff
@@ -1356,6 +1393,41 @@ int LAGr_MaximumMatching(
                    // the columns' or from the rows' perspective, ignored if
                    // mate_init is NULL
     char *msg);
+//------------------------------------------------------------------------------
+// LAGraph_RichClubCoefficient: Compute Rich Club Coefficient of Graph
+//------------------------------------------------------------------------------
+
+LAGRAPHX_PUBLIC
+int LAGraph_RichClubCoefficient
+(
+    GrB_Vector *rich_club_coefficents, //output
+    LAGraph_Graph G, //input graph
+    char *msg
+) ;
+
+//------------------------------------------------------------------------------
+// LAGraph_SwapEdges: Randomize Graph while maintaining degree sequence. 
+//------------------------------------------------------------------------------
+LAGRAPHX_PUBLIC
+int LAGraph_SwapEdges
+(
+    // output
+    GrB_Matrix *A_new, //The adjacency matrix of G with edges randomly swapped
+    // input: not modified
+    LAGraph_Graph G,
+    GrB_Index Q, // Swaps per edge
+    char *msg
+) ;
+
+LAGRAPHX_PUBLIC
+int LG_CC_FastSV7_FA // SuiteSparse:GraphBLAS method, with GxB extensions
```

Comment:
This should not appear here, but in LG_internal.h instead.  It's not meant to be user-callable.